dashboard/db/schema.rb

```diff
@@ -291,6 +291,7 @@
     t.string   "md5"
     t.boolean  "published",                              default: false, null: false
     t.text     "notes",                    limit: 65535
+    t.text     "audit_log",                limit: 65535
```

Comment:
I've been using postgresql for a side project and really enjoying [its native `jsonb` data type](https://www.postgresql.org/docs/9.4/static/datatype-json.html) for stuff like this.  [MySQL added a JSON data type back in 2015](https://dev.mysql.com/doc/refman/5.7/en/json.html), I wonder if that's an option for us.